packages/relay-compiler/bin/RelayCompilerBin.js

```diff
@@ -100,6 +100,14 @@ const argv = yargs
       type: 'string',
       default: null,
     },
+    customScalars: {
+      describe:
+        'Mappings from custom scalars in your schema to built-in GraphQL ' +
+        'types, for type emission purposes. (Uses yargs dot-notation, e.g. ' +
+        '--customScalars.URL=String)',
+      type: 'object',
```

Comment:
@alloy I'm getting flow errors for this type. Is it actually supported? I can't find it in the docs: http://yargs.js.org/docs/#api-optionkey-opt

Reply:
going to merge this in anyway and we can fix in a follow up

Reply:
It doesn’t exist [yet](https://github.com/yargs/yargs/issues/890), in the sense that it checks the input type, but setting it here does make it show up in the help banner and so I think it’s helpful to include already.

I did not notice any flow errors at the time, but also the relay-config PR applies custom typings to these options, so perhaps that overrides any current failures here.

Reply:
got it! thanks! I'll just leave it with a $FlowFixMe for now then